### DISCUSSION TITLE
Is there a way to get alerts when something goes wrong?

### DISCUSSION THREAD

**Discussion by ksdhans** [btcpayserver/btcpayserver#6138] [Ideas / Feature Requests]
Yesterday I discovered that my BTCPay server's bitcoin node had stopped working properly. The first warning I got was a message that a lightning node channel had been force closed.

Is there a way to get alerts (via email, SMS or some other channel) if some part of BTCPay goes offline? I can use an uptime monitor for the HTTP(S) component. However, what about other parts like the bitcoin and lightning network nodes, and NBXplorer? 

No idea why it stopped working. Getting the node resynced is going very slowly, and the UI still says "the node is starting..." even though I can see bitcoind slowly syncing in the logs. I'm pretty sure I'd be up and running faster if I'd caught this earlier...

**Comment by pavlenex** [comment]
Would be good idea if we could add some monitoring hooks for emails eventually.

**Reply by dstrukt** [reply]
ACK

**Comment by pavlenex** [comment]
According to @TChukwuleta this use-case is now possible through https://plugin-builder.btcpayserver.org/public/plugins/serveralert plugin. @ksdhans  please give it a try when you guys have a chance and let @TChukwuleta know how it works and if it covers your needs through this plugin.
